store/layer_generators/main.go

```diff
@@ -17,8 +17,15 @@ import (
 	"text/template"
 )
 
+const OPEN_TRACING_PARAMS_MARKER = "@openTracingParams"
+
 func main() {
-	code := GenerateTimerLayer()
+	BuildTimerLayer()
+	BuildOpenTracingLayer()
+}
+
+func BuildTimerLayer() {
```

Comment:
Similar comments as the other main function.
- Unexport functions and structs.
- Remove panics.